ksql-parser/src/main/java/io/confluent/ksql/parser/properties/with/PropertiesConfig.java

```diff
@@ -69,6 +74,25 @@ public String toString() {
     return new HashMap<>(originalLiterals);
   }
 
+  public Optional<ColumnRef> getTimestampColumn() {
```

Comment:
This class should not _know_ about any specific property. It is a generic helper in the same way that we wouldn't put knowledge of specific properties into `ConfigDef` or `AbstractConfig`.

Better to move this into a utility class that both properties impls use, or maybe `CommonCreateConfigs`?

Reply:
My next PR moves this code out anyway, so I'll couple it with that (I do what's in the comment)